core/src/main/java/io/questdb/cairo/RebuildColumnBase.java

```diff
@@ -281,6 +283,10 @@ private void reindexOneOrAllColumns(
                     );
                 }
             }
+            //To validate if the table has any index or not
+            if (isIndexed) {
```

Comment:
We definitely don't want to return an error if the table has indexes.
```suggestion
            if (!isIndexed) {
```

Reply:
Will make the changes.